src/app/docs/input-box/input-box-docs.component.html

```diff
@@ -0,0 +1,222 @@
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Hover
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-hover.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Focus
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-focus.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell sky-docs-table-cell-primary">
+            Click
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-click.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Disabled
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-disabled.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Required
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-required.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+        <tr>
+          <td class="sky-docs-table-cell">
+            Error
+          </td>
+          <td class="sky-docs-table-cell">
+            <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/input-box/input-box-states-error.png"
+            >
+            </sky-docs-design-guideline-thumbnail>
+          </td>
+        </tr>
+      </table>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Make the input label succinct and descriptive. Don't construct sentences or ask questions with the input label. Put conversational labels or lengthy descriptions outside the Input box and use section headings. See  <a href="https://developer.blackbaud.com/skyux/design/guidelines/form-design">Form design</a> guidelines for more information.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          caption="Do use section headers outisde of Input box to supply conversational form design."
```

Comment:
I'd add "the" after "of" and lowercase "input."

Reply:
Done.

Reply:
Dang! Just caught that Denise and I both missed the misspelled `outside` on our first review!!! I'll fix that now...